src/site/xdoc/index.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document>
+    <properties>
+        <title>Flexmark Library</title>
+        <author>Lutece Team</author>
+        <version>1.0.0</version>
+    </properties>
+    <body>
+        <section name="Introduction">
+            <p>
+                The Flexmark library provides functionalities to parse and render Markdown within the Lutece platform.
+            </p>
+        </section>
+        <section name="Configuration">
+            <p>
+                To configure the Flexmark library, follow these steps:
+            </p>
+            <itemizedlist>
+                <item>Include the library in your project dependencies.</item>
+                <item>Initialize the FlexmarkService in your application.</item>
+            </itemizedlist>
+        </section>
+        <section name="Usage">
+            <p>

```

Comment:
Please add : "This plugin delivers  an implementation of the IRichTextContentParser interface,  used by the Lutece RichTextContentService. The content to parse shall begin with the prefix specified in the injection configuration."